cmd/update-index/main.go

```diff
@@ -261,25 +264,33 @@ func main() {
 	}); err != nil {
 		panic(err)
 	}
-	ioutil.WriteFile(args.outputFile, buf.Bytes(), os.FileMode(0644))
+
+	err = ioutil.WriteFile(args.outputFile, buf.Bytes(), os.FileMode(0644))
+	if err != nil {
+		panic(err)
```

Comment:
Do we want to panic here or maybe just return the error instead?
(Same comment for the other usages of `panic`.)

Reply:
same comment as above. wdyt?